itest/lnd_mpp_test.go

```diff
@@ -558,7 +560,35 @@ func (m *mppTestScenario) buildRoute(amt btcutil.Amount,
 		FinalCltvDelta: chainreg.DefaultBitcoinTimeLockDelta,
 		HopPubkeys:     rpcHops,
 	}
-	routeResp := sender.RPC.BuildRoute(req)
 
-	return routeResp.Route
+	// We should be able to call `sender.RPC.BuildRoute` directly, but
+	// sometimes we will get a RPC-level error saying we cannot find the
+	// node index:
+	// - no matching outgoing channel available for node index 1
+	// This happens because the `getEdgeUnifiers` cannot find a policy for
+	// one of the hops,
+	// - [ERR] CRTR router.go:1689: Cannot find policy for node ...
+	// However, by the time we get here, we have already checked that all
+	// nodes have heard all channels, so this indicates a bug in our
+	// pathfinding, specifically in the edge unifier.
+	//
+	// TODO(yy): Remove the following wait and use the direct call, then
+	// investigate the bug in the edge unifier.
+	var route *lnrpc.Route
```

Comment:
so eventually it shows up after some time the missing edge ?

Reply:
yeah, which is weird, think it's most likely due to graph being cached and not updated in time.